shell/profiling/sampling_profiler.h

```diff
@@ -29,15 +29,29 @@ struct CpuUsageInfo {
   double total_cpu_usage;
 };
 
+/**
+ * @brief Memory usage stats. `dirty_memory_usage` is the number of mega bytes
+ * such that the app uses its physical memory for dirty memory. Dirty memory
+ * is the memory data that cannot be paged to disk. `owned_share_memory_usage`
+ * is the number of mega bytes such that the app uses its physicaal memory for
+ * shared memory, including loaded frameworks and executables. On iOS, it's
+ * `physical memory - dirty memory`.
+ */
+struct MemoryUsageInfo {
+  double dirty_memory_usage;
+  double owned_share_memory_usage;
```

Comment:
rename this to `owned_shared_memory_usage`.

Reply:
Done.